doc/contributing.rst

```diff
@@ -175,12 +175,18 @@ Get the `docker-compose.yml` configuration file, it will automatically be filled
 
 
 * set `SITE_URL` to `https://localhost:3148` or whatever port you chose in the previous step.
-* Change the `volumes:` line so it points to your `$dev/elabftw` folder (for elabftw and mysql containers). The part before the `:` is the path on your host, that you should adjust, and the part after corresponds to where the code lives in the container: `/elabftw`, and this you do not change. It should look like this:
+* Change the `volumes:` lines to bind mount the container to the source code. The part before the `:` is the source path on your host that you should adjust. The part after corresponds to the destination path where the code lives in the container. For the elabftw container, adjust the source to point to the `$dev/elabftw` path and adjust the destination path to `/elabftw`. For the mysql container, adjust the source to point to `$dev/elabftw/mysql` and keep the destination path as `/var/lib/mysql`. The lines should look like this:
```

Comment:
I think it can be worthwhile to extend that section and be clearer, see for instance:

https://chat.openai.com/share/28480158-02d0-41b0-88e4-82fbf7c3c078